CBLAS/src/CMakeLists.txt

```diff
@@ -120,5 +120,9 @@ set_target_properties(
   VERSION ${LAPACK_VERSION}
   SOVERSION ${LAPACK_MAJOR_VERSION}
   )
+target_include_directories(cblas PUBLIC
```

Comment:
> I think INCLUDES DESTINATION is also a v2.8.12 feature

Yes, that's what I meant by "If update to 2.8.12 is acceptable, alternatively..." :wink: